Build/downgrade_nsis.ps1

```diff
@@ -2,16 +2,45 @@
 ##  File:  Install-NSIS.ps1
 ##  Desc:  Install NSIS
 ################################################################################
+function Set-MachinePath{
```

Comment:
including all cmd-lets for updating the path. taken from https://www.powershellgallery.com/packages/CircleCIDSC/1.0.1098/Content/ExamplePacker%5Cwindows%5CImageHelpers%5CPathHelpers.ps1

Reply:
this could probably done easier... asking ChatGPT now